rust/src/lib.rs

```diff
@@ -367,3 +386,11 @@ fn get_s3_data_keys(name: &str) -> [String; 3] {
         format!("{name}.meta"),
     ]
 }
+
+/// Return possible env variable value as Option
+fn get_env_variable(name: &str) -> Option<String> {
+    match env::var(name) {
+        Ok(value) => Some(value),
+        Err(_) => None,
+    }
+}
```

Comment:
```suggestion
fn get_env_variable(name: &str) -> Option<String> {
    env::var(name).ok()
}
```
This does the same thing?